### ISSUE TITLE
Icon too big on nested context menu

### ISSUE THREAD

**Issue by Fantazzj** [oleg-shilo/shell-x#12]
When I put an icon for the nested context menu it appears to big.

The problem
![immagine](https://user-images.githubusercontent.com/63418260/161995278-42148afd-4b38-4698-a255-1dd50d39ee8a.png)

The folder of the configuration:
![immagine](https://user-images.githubusercontent.com/63418260/161995527-6cf0aeec-aa29-47ad-8bf2-196c5580a6c8.png)

I'm using the one it is included with the software, i've tried even with different icons from 32x32 and bigger, but the same happens.
I'm ond windows 10 and used chocolatey to install shell-x.

**Comment by oleg-shilo** [comment]
Not sure I have too many options to address it.

The icon file is taken as is and assigned to the standard .NET WinForms menu item:
```C#
var menu = new ToolStripMenuItem
           {
               Text = item.ToFileMenuText(),
               Image = LookupImageFor(item)
           };
``` 
Thus the extension itself does not participate in the menu rendering. Thus there is no mechanism to change the rendering algorithm.

I am guessing there is something tricky with your monitor resolution and scaling. Under normal circumstances, as you can see the very same icon works just fine:

![image](https://user-images.githubusercontent.com/16729806/162191108-42788a2d-f969-48df-b823-8cf2e09fb8f7.png)

Not sure what can be done if you have to keep your calling. Maybe you can just develop your custom icon if the benefit is worth the effort.

**Comment by oleg-shilo** [comment]
Sorry for the late response. Indeed there was a mistake in the image assignment algorithm for folders.
Fixed in the Release v1.5.2
